### PR TITLE
Updated to work with 0.30.1

### PR DESCRIPTION
**Pull request by watzon**
Awesome project. Saw that you were using ngrok.cr and figured I'd give you a hand getting this up to date.

### CODE REVIEW

File: src/quicksand/sand_handler.cr [hunk 1]
```diff
@@ -23,8 +23,8 @@ class Quicksand::SandHandler
     when "/"
       context.response.status_code = 302
       context.response.headers["Location"] = "/" + file_path
-    when "/" + file_path
-      context.response.headers["Etag"] = %{W/"#{mod_time.epoch}"} if mod_time
+    when file_path
```
Comment (Willamin) [inline src/quicksand/sand_handler.cr]: This change causes the program to not work for me; is there a reason you made this change?

**Review by Willamin — CHANGES_REQUESTED** [review]

File: src/quicksand/sand_handler.cr (hunk 1 above)
Comment (watzon) [reply src/quicksand/sand_handler.cr]: It should work, let me test again. On my system it was adding an extra `/` which was messing things up.

Comment (watzon) [reply src/quicksand/sand_handler.cr]: Yeah I just re-tested, with the `/` beforehand it doesn't work on either of the systems I've tested it on. Something that should work is if I do `File.join("/", file_path)`. I can push that update quickly.

Comment (Willamin) [reply src/quicksand/sand_handler.cr]: Awesome!

**Comment by Willamin** [comment]
@watzon Thank's so much for contributing!

**Pull request merged**